tests/test_randomizer.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+import dice
+import randomizer
+
+
+def test_add_ability_to_pool(ability_d4):
+    randomizer.add_ability_to_pool(ability_d4)
+    assert len(randomizer.Pool) == 1
+
+
+def test_remove_ability_from_pool(ability_d4):
+    initial_len = len(randomizer.Pool)
```

Comment:
Is it possible that `initial_len` is zero? This is the case with global objects: you should avoid them. And with tests, you should always clear your environment's state for every test.